EMACS.md

```diff
@@ -0,0 +1,24 @@
+# Running import-js in Emacs
+
+1. Install import-js
+  * `gem install import-js`
+2. Configure import-js
+  * See [Configuration](README.md#configuration)
+2. Install import-js.el for Emacs
+  * This is not yet available on elpa/melpa, so for now this means dropping
+    import-js.el somewhere in your .emacs.d
```

Comment:
Well right now its not being packaged in the gem, so the only place to get it is the repo. The much better way to do this is publish it on melpa, but also want it stable first.